src/VerifyWindowsHelloController.js

```diff
@@ -94,22 +113,88 @@ function (Okta, FormController, FormType, webauthn, FooterSignout, WindowsHelloE
             )
           );
         }
+        else {
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-default">\
+              {{i18n code="verify.windowsHello.subtitle" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: null}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-loading">\
+              {{i18n code="verify.windowsHello.subtitle.loading" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'loading'}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-success">\
+              {{i18n code="verify.windowsHello.subtitle.signingIn" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'success'}
+          }));
+        }
+
+        result.push(FormType.View({View: Spinner}));
+
         return result;
       },
 
+      postRender: function () {
+        if (this.options.appState.get('factors').length === 1 && !this.model.get('__enrollmentState__')) {
+          this.model.save();
```

Comment:
After thinking about it a little more, it seems to me that this is something that we will need to add, but not as simple. So for now I think we should leave it as it is (open the modal when WindowsHello is the only factor), and if there are more factors then show the button.